packages/plugin-schema-validator/src/test/kotlin/ly/iterative/itly/SchemaValidatorPluginTest.kt

```diff
@@ -63,7 +63,7 @@ class SchemaValidatorPluginTest {
     @Test
     fun validate_invalidEventWithErrorOnInvalid_throwsError() {
         val itly: Itly = TestUtil.getItly(Context.VALID_ALL_PROPS, Options(
-            plugins = arrayListOf(SchemaValidatorPlugin(Schemas.DEFAULT)),
+            plugins = arrayListOf(SchemaValidatorPlugin(Schemas.DEFAULT) as AnyPluginType),
```

Comment:
@dantetam this still produces warnings. Can you please try to find a better solution. 

`w: /Users/justin-fiedler/Documents/dev/itly-sdk-jvm/packages/plugin-schema-validator/src/test/kotlin/ly/iterative/itly/SchemaValidatorPluginTest.kt: (66, 74): Unchecked cast: SchemaValidatorPlugin to AnyPluginType /* = Plugin<PluginCallOptions, PluginCallOptions, PluginCallOptions, PluginCallOptions> */`